harbor-client/src/lib.rs

```diff
@@ -1035,15 +1400,46 @@ impl HarborCore {
         Ok(())
     }
 
-    pub async fn get_federation_items(&self) -> Vec<FederationItem> {
+    pub async fn remove_cashu_mint(&self, _msg_id: Uuid, url: &MintUrl) -> anyhow::Result<()> {
+        log::info!("Removing cashu mint: {url}");
+
+        // Cancel any ongoing metadata fetch
+        self.metadata_fetch_cancel.store(true, Ordering::Relaxed);
+
+        // Small delay to allow any in-progress operations to complete
```

Comment:
this seems weird but I trust u

Reply:
This was in the remove fedimint stuff so I just kept it